freqtrade/arguments.py

```diff
@@ -171,6 +187,46 @@ def backtesting_options(parser: argparse.ArgumentParser) -> None:
             metavar='PATH',
         )
 
+    @staticmethod
+    def edge_options(parser: argparse.ArgumentParser) -> None:
+        """
+        Parses given arguments for Backtesting scripts.
+        """
+        parser.add_argument(
+            '-r', '--refresh-pairs-cached',
```

Comment:
i think we should extract the common options between edge / backtesting (but not hyperopt) to a seperate function to avoid code duplication ... now we have this (and the 2 export options" duplicated ...)?